Resources/Views/Home/_sponsors.leaf

```diff
@@ -6,9 +6,20 @@
     </h2>
   </div>
 
-  <div class="col-sm-7 col-md-9 col-lg-10">
-    <p>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-    </p>
-  </div>
+    <div class="table-responsive">
+      <table class="table text-center">
+        <tbody>
+          <tr>
+            <th scope="row" class="text-start">Platinum Sponsors</th>
+            <td><a href="https://codemagic.io"><img src="img/codemagic.png" class="sponsor-img"></a></td>
```

Comment:
These should probably have target="_blank" on them so they open in new tabs rather than replacing our site